app/src/test/java/com/codesoom/assignment/controllers/TaskControllerTest.java

```diff
@@ -0,0 +1,34 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.models.Task;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import static org.assertj.core.api.Assertions.*;
+
+class TaskControllerTest {
+    private TaskController controller;
+
+    @BeforeEach
+    void setUp() {
+        controller = new TaskController();
+    }
+
+    @Test
+    void list() {
+         assertThat(controller.list()).isEmpty();
+
+    }
+
+    @Test
+    void createNewTask() {
+        Task task = new Task();
+        task.setTitle("Test");
+
+        controller.create(task);
+
+        assertThat(controller.list()).hasSize(1);
+        assertThat(controller.list().get(0).getId()).isEqualTo(1L);
+        assertThat(controller.list().get(0).getTitle()).isEqualTo("Test");
+    }
+
+}
```

Comment:
파일 마지막에 공백 라인이 없어 경고가 나오고 있습니다. 공백 라인을 추가해 주세요. 
```suggestion
}

```